src/rust/dbv2/migrations/2023-09-20-065710_event_types/up.sql

```diff
@@ -0,0 +1,89 @@
+-- Your SQL goes here
+CREATE TABLE
+  change_order_size_events (
+    txn_version NUMERIC(20) NOT NULL,
+    event_idx NUMERIC(20) NOT NULL,
+    PRIMARY KEY (txn_version, event_idx),
+    market_id NUMERIC(20) NOT NULL,
+    TIME timestamptz NOT NULL,
+    order_id NUMERIC(39) NOT NULL,
+    "user" VARCHAR(70) NOT NULL,
+    custodian_id NUMERIC(20) NOT NULL,
+    side BOOLEAN NOT NULL,
+    new_size NUMERIC(20) NOT NULL
+  );
+
+
+CREATE FUNCTION notify_change_order_size_event () RETURNS TRIGGER AS $$
+BEGIN
+   PERFORM pg_notify('change_order_size_event'::text, row_to_json(NEW)::text);
+   RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+
+CREATE TRIGGER change_order_size_events_trigger
+AFTER INSERT ON change_order_size_events FOR EACH ROW
+EXECUTE PROCEDURE notify_change_order_size_event ();
```

Comment:
Each of these needs something to this effect after the CREATE TRIGGER:

```
CREATE VIEW api.fill_events AS SELECT * FROM fill_events;

GRANT SELECT ON api.fill_events TO web_anon;
```

So that `web_anon` can see the events.

Reply:
Done